components/VcDetails.tsx

```diff
@@ -352,6 +353,18 @@ function getLocalizedField(rawField: string | LocalizedField[]) {
   }
   try {
     const locales: LocalizedField[] = JSON.parse(JSON.stringify(rawField));
+    const supportedLanguages = Object.keys(SUPPORTED_LANGUAGES);
+    const currentLanguage = i18n.language;
+    if (locales.length > 1) {
```

Comment:
Since We are using the same code snippet in both places, can we extract a method and use that